### DISCUSSION TITLE
Colossal Candy Clutter

### DISCUSSION THREAD

**Discussion by Bear64DX** [EmeraldLoc/tag#620] [Q&A] [answered]
Ever since I made my suggestion on removing MMM, I've come to notice that there are a lot of similarities about the 2 stages (which isn't a good thing). For the longest time I didn't have any plans to remove this stage, however they are really similar. And I'm still mixed on this stage. So I was wondering if you guys had any ideas for this stage, whether it should stay, change a bit, or get removed.

**Comment by EmeraldLoc** [comment, marked as answer]
What, this stage is great!

**Reply by Bear64DX** [reply]
Thinking back of what I said, I think it's a decent stage for Tag. WAAAAAAAAAY better than MMM
